tests/perf-test/PolicyUtils.perf-test.ts

```diff
@@ -0,0 +1,33 @@
+import {measureFunction} from 'reassure';
+import {getMemberAccountIDsForWorkspace} from '@libs/PolicyUtils';
+import type {PersonalDetails, PolicyMember} from '@src/types/onyx';
+import createCollection from '../utils/collections/createCollection';
+import createPersonalDetails from '../utils/collections/personalDetails';
+import createRandomPolicyMember from '../utils/collections/policyMembers';
+
+describe('PolicyUtils', () => {
+    describe('getMemberAccountIDsForWorkspace', () => {
+        test('500 policy members with personal details', async () => {
+            const policyMembers = createCollection<PolicyMember>(
+                (_, index) => index,
+                () => createRandomPolicyMember(),
+            );
+            const personalDetails = createCollection<PersonalDetails>((_, index) => index, createPersonalDetails);
+
+            await measureFunction(() => getMemberAccountIDsForWorkspace(policyMembers, personalDetails));
+        });
+
```

Comment:
I am a bit weary on including both of these tests since if the member has any error they will not be returned. 

Not a big deal, the function is not that complex so we could include both cases I think